region/lowernorfair/east.json

```diff
@@ -3502,48 +5037,111 @@
           "from": 5,
           "to": [
             {
-              "id": 2,
-              "requires": [
-                {"or": [
-                  "canUseSpringBall",
-                  "canUseMorphBombs",
-                  {"obstacle":{
-                    "id": "A",
-                    "requires": [ 
-                      {"enemyKill":{
-                        "enemies": [
-                          [ "Fune" ]
-                        ]
-                      }}
-                     ]
-                  }}
-                ]},
-                {"or": [
-                  "canWalljump",
-                  "canFly",
-                  {"and": [
-                    "HiJump",
-                    {"or": [
-                      "SpeedBooster",
-                      "canSpringBallJumpMidAir"
+              "id": 6,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": true,
+                  "requires": [
+                    {"or":[
+                      "canFly",
+                      {"and": [
+                        "HiJump",
+                        {"or": [
+                          "SpeedBooster",
+                          "canSpringBallJumpMidAir",
+                          "canWalljump"
```

Comment:
I left out strats that can get you up to the claw but not up to door level (without walljumping).

I think maybe it would be sensible to add one more junction on the claw, then we could separate the two jumps.

Reply:
Pb boost into bugboost requires hjb, right? I'd consider putting that in for shits and giggles (maybe could be useful waaaaaay down the line for some non-walljump hack or something) if you could get more than one shot at pulling it off.

Reply:
This highjump+pb boost into bug boost is a meme, but the highjump+dboost should be added.  Its consistent with some practice.